github_rules/github_branch_protection_disabled.py

```diff
@@ -0,0 +1,9 @@
+def rule(event):
+    return event.get("action") == "protected_branch.destroy"
+
+
+def title(event):
+    return (
+        f"A branch protection was removed from the "
+        f"repository [{event.get('repo', '<UNKNOWN_REPO>')}]"
```

Comment:
And I don't suppose we're able to get the name of the branch for which branch protection was removed? I assume not based on your comments, but just double checking.

Reply:
nope :(